cmd/prometheus/main.go

```diff
@@ -1430,7 +1433,7 @@ func reloadConfig(filename string, expandExternalLabels, enableExemplarStorage b
 
 	noStepSuqueryInterval.Set(conf.GlobalConfig.EvaluationInterval)
 	l := []interface{}{"msg", "Completed loading of configuration file", "filename", filename, "totalDuration", time.Since(start)}
-	level.Info(logger).Log(append(l, timings...)...)
+	logger.Info(fmt.Sprintf("%v", append(l, timings...)...))
```

Comment:
This probably needs reworking for slog; it's trying to pass paired name/values.
Perhaps `[]slog.Attr` ?